presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -246,6 +248,19 @@ protected Node visitShowGrants(ShowGrants showGrant, Void context)
         }
 
         @Override
+        protected Node visitShowRoles(ShowRoles node, Void context)
+        {
+            if (!node.getCatalog().isPresent() && !session.getCatalog().isPresent()) {
+                throw new SemanticException(CATALOG_NOT_SPECIFIED, node, "Catalog must be specified when session catalog is not set");
+            }
+
+            String catalog = node.getCatalog().orElseGet(() -> session.getCatalog().get());
+            return simpleQuery(
+                    selectList(aliasedName("role_name", "Role")),
```

Comment:
I actually like the alias because it makes the column header Role which is much neater than role_name. (and similar to how SHOW TABLES and SHOW SCHEMAS works)